diff --git a/node_modules/@fubei/web-fetch/dist/index.js b/node_modules/@fubei/web-fetch/dist/index.js
index 2adab35..1a0afc5 100644
--- a/node_modules/@fubei/web-fetch/dist/index.js
+++ b/node_modules/@fubei/web-fetch/dist/index.js
@@ -465,6 +465,7 @@
         // web端 aixos请求拦截
         WebHttpService.prototype.requsetInterceptor = function () {
             this.axios.interceptors.request.use(function (config) {
+              console.log('请求拦截：', config)
                 return config;
             }, function (error) {
                 return Promise.reject(error);
@@ -483,6 +484,7 @@
          * @param {object} reqOptions 请求参数 {object} reqData 处理之后的参数, {number} retryTimes 重试次数, {promise.resolve} resolve, {promise.reject} reject
          */
         WebHttpService.prototype._request = function (reqOptions, reqData, context, resolve, reject) {
+            console.log('web端请求开始了', reqOptions)
             var _this = this;
             var axiosInstance;
             var baseUrl = this.getCurrentUrl(reqOptions);
@@ -490,6 +492,7 @@
             context = __assign(__assign({}, context), { baseUrl: baseUrl });
             if (reqOptions.method === 'GET') {
                 axiosInstance = this.axios.get(baseUrl, {
+                    headers: getHeader(reqOptions.header, this.defaultHeader),
                     params: reqOptions.data,
                     timeout: timeout
                 });
